docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+# Repository Management
+
+The goal is for this to be a living document explaining how we collaboratively manage the Gutenberg repository. If you’d like to suggest a change, please open an issue for discussion or submit a pull request to the document.
+
+This document covers:
+
+- [Issues](#issues)
+  - [Labels](#labels)
```

Comment:
These links don't work because the Markdown bolding `**` don't assign `id`. Might want to use a nested heading level instead, i.e. `###`

Reply:
Good catch cae7bd474d33319eecec5b8396e6c84e793f5f1a